### PR TITLE
chore: Pull request triage automation:

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change?

* Add new pull request to the *Reviews* project in the *Triage* column. See: https://github.com/eclipse/che-docs/projects/1
* Move assigned pull request to the *In progress* column.

## What issues does this pull request fix or reference?

This task doesn't need to be manually done.

### CODE REVIEW

**Review by l0rd — APPROVED** [review]

**Comment by themr0c** [comment]
We want to use more the project to help visualize the state of each pull request https://github.com/eclipse/che-docs/projects/1

Current vs. change:
Assigning reviewers  to an opened  pull request is automatic - we keep it.
Assigning an opened pull request to the project is manual - we automate it.
Assigning the writer (in the assignee field, not in the reviewer field) is manual - we keep it.
Moving from "Triage" to "In progress"  when the pull request receives an assignee is manual - we automate it.
Moving from "In progress" to "Engineering review in progress" is manual - we keep it.

File: .github/workflows/pull-request-assigned.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Automate pull requests triage
+
+name: Pull request assigned
+
+on:
+  pull_request:
```
Comment (benoitf) [inline .github/workflows/pull-request-assigned.yml]: it won't work for forks as forks won't have access to secrets with "pull_request" event

File: .github/workflows/pull-request-opened.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Automate pull requests triage
+
+name: Pull request opened
+
+on:
+  pull_request:
```
Comment (benoitf) [inline .github/workflows/pull-request-opened.yml]: it won't work for forks as forks won't have access to secrets with "pull_request" event

File: .github/workflows/pull-request-assigned.yml (hunk 1 above)
Comment (themr0c) [reply .github/workflows/pull-request-assigned.yml]: Oh :(. Nice spot. Is there a way to circumvent it other than creating 2 workflows, like for the PR preview publication?

Comment (benoitf) [reply .github/workflows/pull-request-assigned.yml]: for this one you can use `pull_request_target` event as it's not using user data so there is no security risk

Comment (themr0c) [inline .github/workflows/pull-request-assigned.yml]:
```suggestion
  pull_request_target:
```

File: .github/workflows/pull-request-opened.yml (hunk 2 above)
Comment (themr0c) [inline .github/workflows/pull-request-opened.yml]:
```suggestion
  pull_request_target:
```

**Review by benoitf — APPROVED** [review]

**Pull request merged**